charts/istio/istio-ingress/templates/envoy-filter.yaml

```diff
@@ -22,10 +22,78 @@ spec:
     match:
       context: GATEWAY
       cluster:
-        portNumber: {{ .port }}
+        portNumber: {{ .targetPort }}
     patch:
       operation: MERGE
       value:
         per_connection_buffer_limit_bytes: 32768 # 32 KiB
 {{- end }}
 {{- end }}
+
+---
+
+apiVersion: networking.istio.io/v1alpha3
+kind: EnvoyFilter
+metadata:
+  name: tcp-stats-filter-1.7
```

Comment:
A bit strange name. Maybe a short description via annotations will be useful.

Reply:
This is the same name as the upstream one https://github.com/istio/istio/blob/876e4a967883e600bc4bd544c0dea20309437faa/manifests/charts/istio-control/istio-discovery/templates/telemetryv2_1.7.yaml#L117